homeassistant/components/vilfo/__init__.py

```diff
@@ -69,9 +69,10 @@
 class VilfoRouterData:
     """Define an object to hold sensor data."""
 
-    def __init__(self, host, access_token):
+    def __init__(self, host, access_token, hass):
```

Comment:
Please put `hass` first in the signature. That's our standard whenever we need to pass `hass`.

Reply:
:+1: Done! I'll add that to that list of things to add to the developer documentation (at least I haven't seen it there) 😄